discovery/src/test/java/com/ibm/watson/developer_cloud/discovery/v1/DiscoveryServiceIT.java

```diff
@@ -131,11 +104,10 @@ public static void setupClass() {
     Assume.assumeFalse("config.properties doesn't have valid credentials.",
         (username == null) || username.equals(PLACEHOLDER));
 
-    GetEnvironmentsRequest getRequest = new GetEnvironmentsRequest.Builder().build();
-    GetEnvironmentsResponse getResponse = discovery.getEnvironments(getRequest).execute();
+    GetEnvironmentsResponse getResponse = discovery.getEnvironments(null).execute();
     for (Environment environment : getResponse.getEnvironments()) {
       // look for an existing environment that isn't read only
-      if (!environment.isReadOnly()) {
+      if (!environment.getReadOnly()) {
```

Comment:
so we can't change it so that getters of booleans read as `is<variable>` instead of `get<variable>`?